micrometer/src/main/java/feign/micrometer/DefaultFeignObservationConvention.java

```diff
@@ -48,14 +49,16 @@ public String getContextualName(FeignContext context) {
 
   @Override
   public KeyValues getLowCardinalityKeyValues(FeignContext context) {
-    String templatedUrl = context.getCarrier().requestTemplate().methodMetadata().template().url();
+    RequestTemplate requestTemplate = context.getCarrier().requestTemplate();
```

Comment:
Hey guys, just came across this PR, as I needed to include the client name and was writing a similar implementation, but noticed something in this change. Isn't `context.getCarrier()` nullable and shouldn't we check for `null` like it is done in `getMethodString(@Nullable Request request)` a bit further down?